core/src/test/scala/org/apache/spark/util/collection/PercentileHeapSuite.scala

```diff
@@ -21,46 +21,67 @@ import java.util.NoSuchElementException
 
 import org.apache.spark.SparkFunSuite
 
-class MedianHeapSuite extends SparkFunSuite {
+class PercentileHeapSuite extends SparkFunSuite {
 
-  test("If no numbers in MedianHeap, NoSuchElementException is thrown.") {
-    val medianHeap = new MedianHeap()
+  test("If no numbers in PercentileHeap, NoSuchElementException is thrown.") {
+    val medianHeap = new PercentileHeap()
```

Comment:
Shall we rename the variable name too, `val medianHeap` -> `val percentileHeap`?